native/exe/SageLauncher/launcher.cpp

```diff
@@ -626,7 +626,7 @@ int WINAPI WinMain( HINSTANCE hInst, 	/*Win32 entry-point routine */
 	LPTSTR folderPath = new TCHAR[2048];
 	GetModuleFileName(NULL, folderPath, 2048);
 	size_t appLen = strlen(folderPath);
-	for (size_t i = appLen - 1; i > 0; i--)
+	for (size_t i = appLen; i-- > 0;) // this does the check for continuation on the value before decrement, but still uses the decremented value inside the loop 

```

Comment:
Ditto

Reply:
Changed to  'if (appLen > 0)' conditional